sdk/requirements/python-3-7.txt

```diff
@@ -1,7 +1,7 @@
 setuptools<=67.2.0
 cloudpickle<=2.2.0
 numpy<=1.24.0
-requests<=2.28.2
+requests>=2.28.1,<=2.28.2
```

Comment:
Do we have any concerns about restricting the version to a very narrow range? @cw75 @kenxu95

Reply:
I feel like it's ok as a very temporary solution but we should definitely look into resolving this. What I don't understand is why these bounds are here in the first place? @likawind can you add some documentation for what packages `requests` collides with when we stray outside these bounds (in both upper and lower directions)? I'm not sure I'm seeing it from your PR description.

Reply:
@kenxu95 We added the upper-bounds in case the latest versions have some backward-incompatible changes. We should periodically test and bump these upper-bounds, so that they are not lagging behind a major release.

Reply:
ok looks like the upper bound is the latest version. Can we still document the conflict for the lower bound?